features/admin.tenants.v1/constants/tenant-constants.ts

```diff
@@ -86,4 +88,18 @@ export default class TenantConstants {
             TENANT_DELETION: "tenants.deletion"
         };
 
+    public static readonly AUTH_TYPES: AuthenticationTypeDropdownOption[] = [
+        {
+            key: AuthenticationType.NONE,
+            text: "console:manage.features.serverConfigs.remoteLogPublishing.fields.advanced." +
+							"basicAuthConfig.types.none.name",
```

Comment:
Anything preventing us from keeping this in a single line?

Reply:
Yes, this line exceeds the character length